apps/test/unit/lib/tools/jsinterpreter/JSInterpreterTest.js

```diff
@@ -7,6 +7,79 @@ import JSInterpreter from '@cdo/apps/lib/tools/jsinterpreter/JSInterpreter';
 describe('The JSInterpreter class', function() {
   var jsInterpreter;
 
+  describe('static function getFunctionsAndMetadata', () => {
```

Comment:
This test coverage is great!
![chefkiss](https://user-images.githubusercontent.com/2959170/65733072-c3afeb80-e081-11e9-81ed-8560c13e4a1d.gif)